src/appshell/qml/NotationPage/NotationPage.qml

```diff
@@ -320,6 +331,9 @@ DockPage {
             minimumHeight: root.horizontalPanelMinHeight
             maximumHeight: root.horizontalPanelMaxHeight
 
+            maximumWidth: root.horizontalPanelMaxWidth
```

Comment:
I don't think a maximum width is desired for horizontal panels. I'm also afraid that it might cause problems when the panel is docked, because then of course the width depends on the width of the main window, which is definitely more than 800 px.

So I think you can just remove this `maximumWidth:` line, as well as the `root.horizontalPanelMaxWidth` property.

Reply:
I added the maximum width, because if I didn't have it, after making the window floating, it would automatically have a fixed width, which is very small.

Reply:
Hm... that's unfortunate, and surprising. If you have the chance to investigate why this is happening, that would be great; otherwise, I will investigate this myself in the future. It might take a while until I have time for that, unfortunately. Also, some big changes under the hood for the docking framework are planned, which may also affect this problem, so it might make sense to wait investigating until after those changes.